### DISCUSSION TITLE
Shepherd Magic Tree Ideabook

### DISCUSSION THREAD

**Discussion by AsciiSquid** [shiptest-ss13/Shiptest#247] [Ideasguy]
# Shepherd Progression Tree
You know that cool/cursed tree in the middle of the Shepherd-Class ship. It has mechanics, and it *hungers for more*.
Suggest ideas on magic tree progression below.

**Comment by triplezeta** [comment]
oh i forgot to write in this

anyways: have different potential "paths" depending on what chems are poured in other than the gilded and the crimson tree we have now

base these off of terraria trees

make the crimson tree into a different tree whose gimmick is regeneration and lifesteal, make the regular evil tree the corruption one instead :)

**Reply by Sun-Soaked** [reply]
destroying the tree should be really tough, but spawn a poltergeist with a vaguely tree sprite

Tree spirit is angry >:(

**Reply by mcmeiler** [reply]
Destroying the tree creates a Revenant ghostrole

**Reply by triplezeta** [reply]
pouring in napalm and vaguely halloween related reagents turns it into the mourning wood from terraria

it crawls out of the floor and spews fire everywhere 

defeating it drops a stack of spooky wood which lets you craft armor and stuff

**Comment by AtlasRules** [comment]
make it start as a sapling, allowing it to be planted in either dirt, a plant pot or a hydroponics bed
dirt: regular magic tree
plant pot: terracotta warrior tree, earthbending, defensive, blah blah blah
hydroponics bed: tech cult tree, augments and replacing the weak flesh with cool ass robo limbs
